source/npm-util.js

```diff
@@ -61,3 +62,14 @@ exports.prereleaseTags = async packageName => {
 
 	return tags;
 };
+
+exports.checkName = async pkg => {
```

Comment:
It needs a clearer name than `checkName`. Without context I asked myself: "Check the name for what?"